wp-content/themes/core/content/single/post.twig

```diff
@@ -1,32 +1,50 @@
 <article class="item-single">
 
-	{{ post.featured_image }}
-
-	<div class="item-single__content t-content">
-		{{ post.content }}
-	</div>
-
-	<footer class="item-single__footer">
-
-		<ul class="item-single__meta">
-
-			<li class="item-single__meta-date">
+	<div class="item-single__featured-image">{{ post.featured_image }}</div>
+
+	<div class="l-container__stagger item-single__content">
+		<h1 class="item-single__headline">{{ post.title }}</h1>
+		<div class="item-single__meta">
+
+			{% if post.categories %}
+				<div class="item-single__meta__categories">
```

Comment:
Nice job here in terms of the adherence to BEM! One rec for you, to reduce the overall length that can wind up in our BEM class names, instead of the pattern of `parent__child__childofchild__ohnomorechildren`, which is also appropriate, we tend to do something like this: `item-single__meta-categories`, `item-single__meta-categories-list` 😄